jabgui/src/main/java/org/jabref/gui/theme/ThemeManager.java

```diff
@@ -49,13 +55,16 @@ public class ThemeManager {
     );
 
     private static final Logger LOGGER = LoggerFactory.getLogger(ThemeManager.class);
+    private static final ThemeWindowManager THEME_WINDOW_MANAGER = ThemeWindowManagerFactory.create();
```

Comment:
Please avoid creating objects in the static part of a class. It delays application startup and its execution time is not defined. Do this in the constructor.